python/test/t_PlotDesign.py

```diff
@@ -0,0 +1,79 @@
+#! /usr/bin/env python
+
+import os
+import traceback
+import sys
+from matplotlib import pyplot as plt
+
+try:
+
+    # use non-interactive backend
+    import matplotlib
+    if len(sys.argv) == 1:
+        matplotlib.use('Agg')
+
+    from openturns.viewer import PlotDesign
+    import openturns as ot
+    
+    samplesize = 12
+
+    # Plot 2D - basic
+    distribution = ot.ComposedDistribution([ot.Uniform(0.0, 1.0)]*2)
+    bounds = distribution.getRange()    
+    design = distribution.getSample(samplesize)
+    PlotDesign(design);
+
+    # Plot 3D - basic
+    distribution = ot.ComposedDistribution([ot.Uniform(0.0, 1.0)]*3)
+    bounds = distribution.getRange()    
+    design = distribution.getSample(samplesize)
+    PlotDesign(design);
```

Comment:
use `fig = PlotDesign(design)` instead

Reply:
Ok, but why? Does it change anything, since the output argument is not used in this case?